src/Protocol/SmtpPluginManager.php

```diff
@@ -9,22 +9,20 @@
 
 namespace Zend\Mail\Protocol;
 
-use Zend\ServiceManager\AbstractPluginManager;
```

Comment:
Let's have it typehint on container-interop and implement that interface; that'll give the component broader compatibility.